### ISSUE TITLE
Find in page: when toolbar is at top, move "Find in page" to top as well

### ISSUE THREAD

**Issue by davimack** [mozilla-mobile/fenix#22553]
Currently, the URL bar may be configured to appear either at top or bottom of the page. The bar for searching WITHIN a page only appears at the bottom of the page.

The user of the URL bar at the top of their browser may often be confused as to why their URL bar does not show up. The user expectation is that something will be there for them to interact with.

Features to address this could, e.g.:
If the find bar appeared in place of the URL bar, the user would focus where they expect to interact and would then be aware that they needed to close the search bar in order to use the URL bar.

If the application were to recognize aggressive swiping from the top to bottom, it would know that the user is frantically trying to get a URL bar to show up. This could be followed up by a visual indicator that the search bar is open.

If the search bar were colored differently than the rest of the application, that might work?

Long story short:

The search feature behaves differently than the URL bar, and in an unfamiliar way for users who do not search within the page frequently. This feature will be useful to infrequent users of search who have their URL bar at top.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-18709)

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1807141

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
